### PR TITLE
Adds a link to the corresponding GitHub release page in the nav

### PR DESCRIPTION
**Pull request by orta**
I saw this, and was like "Awesome 🌟"

screen shot 2017-01-24 at 10 33 28 am

So I went to the site and tried to click on this:

screen shot 2017-01-24 at 10 27 58 am

and it didn't work, so I figured I could contribute back 💯 

screen shot 2017-01-24 at 10 30 23 am

Which will take you to https://github.com/yarnpkg/yarn/releases/tag/v0.19.1

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -78,12 +78,16 @@
...
+            <a href="https://github.com/yarnpkg/yarn/releases/tag/v{{site.latest_version}}">
+              v{{site.latest_version}}
+            </a>
           </strong>
           {% if site.show_rc %}
             <span aria-hidden="true">&bull;</span>
-            <abbr title="{{i18n.site_nav_rc_version}}">RC</abbr>:
-            <strong>v{{site.latest_rc_version}}</strong>
+              <a href="https://github.com/yarnpkg/yarn/releases/tag/v{{site.latest_version}}">
+                <abbr title="{{i18n.site_nav_rc_version}}">RC</abbr>:
```
Comment (Daniel15) [inline _includes/navigation.html]: Maybe pull this outside of the `<a>`?

Comment (orta) [inline _includes/navigation.html]: Makes sense - done 👍

File: _includes/navigation.html [hunk 2]
```diff
@@ -78,12 +78,18 @@
...
+              v{{site.latest_version}}
+            </a>
           </strong>
           {% if site.show_rc %}
             <span aria-hidden="true">&bull;</span>
-            <abbr title="{{i18n.site_nav_rc_version}}">RC</abbr>:
-            <strong>v{{site.latest_rc_version}}</strong>
+              <abbr title="{{i18n.site_nav_rc_version}}">RC</abbr>:
+              <strong>
+                <a href="https://github.com/yarnpkg/yarn/releases/tag/v{{site.latest_version}}">
```
Comment (Daniel15) [inline _includes/navigation.html]: Sorry, I just missed one thing: This should be `{{site.latest_rc_version}}`

**Comment by orta** [comment]
Ah - my bad! Fixed

**Comment by Daniel15** [comment]
Thanks!

**Pull request merged**